entity-cluster-bot/src/agent_test.py

```diff
@@ -1,108 +1,107 @@
-import agent
+from agent import EntityClusterAgent
 from forta_agent import create_transaction_event
 from datetime import datetime, timedelta
-import os
+import time
 import networkx as nx
 
 from web3 import Web3
 from web3_mock import CONTRACT, EOA_ADDRESS_LARGE_TX, EOA_ADDRESS_NEW, EOA_ADDRESS_OLD, EOA_ADDRESS_SMALL_TX, EOA_ADDRESS_FUNDED_NEW, EOA_ADDRESS_FUNDED_OLD, EOA_ADDRESS_FUNDER_NEW, EOA_ADDRESS_FUNDER_OLD, Web3Mock
 from constants import ALERTED_ADDRESSES_KEY, FINDINGS_CACHE_KEY, GRAPH_KEY
-from forta_agent import get_alerts
+from forta_agent import get_alerts, get_json_rpc_url
+import timeit
+
 w3 = Web3Mock()
+real_w3 = Web3(Web3.HTTPProvider(get_json_rpc_url()))
+from persistance import DynamoPersistance
+
 
 class TestEntityClusterBot:
 
     def remove_persistent_state():
-        if os.path.isfile(ALERTED_ADDRESSES_KEY):
-            os.remove(ALERTED_ADDRESSES_KEY)
-        if os.path.isfile(FINDINGS_CACHE_KEY):
-            os.remove(FINDINGS_CACHE_KEY)
-        if os.path.isfile(GRAPH_KEY):
-            os.remove(GRAPH_KEY)
-    
+        test_util_persistance = DynamoPersistance()
+        test_util_persistance.clean_db()
 
     def test_prune_graph_age(self):
         #  create a graph with some addresses which are older and newer than MAX_AGE_IN_DAYS
         #  assert that the old ones are removed and the new ones are not
         TestEntityClusterBot.remove_persistent_state()
-        agent.initialize()
+        entity_cluster_agent = EntityClusterAgent(DynamoPersistance())
```

Comment:
does the test write to the production DB?

Reply:
yes, will improve: 

1. In s3 it writes to production using a different key to not overwrite production files
2. In the metadata dynamo it uses different tables that is ok
3. it does use the mutex dynamo db of production, that is wrong.

Reply:
Now the test modify DEV database.  It still has both, dev and prod the same s3 but the test save with different bucket key so they don't overwrite